web/styles/settings.css

```diff
@@ -1459,6 +1460,26 @@ $option_title_width: 180px;
     }
 }
 
+#realm_jitsi_server_url_setting {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+
+    select {
+        min-width: auto;
```

Comment:
This `min-width` is eventually computed as `0px` by the browser, so I don't think this is of any use now except that it prevents the dropdown to have width of `325px` which is the `min-width` for all other `select` elements.

Since, we are not sure about the widths currently, I think you can leave it as it is for now, but we should make sure to change it if needed after the UI is finalised to make sure we do not have any unnecessary or weird CSS rules.

Reply:
Maybe add a comment "TODO: This is needed to override the 325px default for dropdowns but ideally we'd remove it." except maybe a bit clearer since I don't actually know what it's for -- but something to help a future developer know to clean it up.

Reply:
Made some changes to the UI and now this is no longer required.